src/hooks.ts

```diff
@@ -18,3 +19,57 @@ export function useFlexSize() {
   const { size } = useContext(boxContext)
   return size
 }
+
+export function useNode() {
+  const { node } = useContext(boxContext)
+  return node
+}
+
+/**
+ * explicitly set the size of the box's yoga node
+ * @requires that the surrounding Flex-Element has `disableSizeRecalc` set to `true`
+ */
+export function useSetSize(): (width: number, height: number) => void {
```

Comment:
Why not
`export function useSetSize(width: number, height: number) {`
?

Reply:
See next comment 😄